tests/conftest.py

```diff
@@ -202,6 +202,21 @@ def do_not_use_colors(monkeypatch):
 """
 
 
+@pytest.fixture
+def iac_single_vuln() -> str:
+    return _IAC_SINGLE_VULNERABILITY
+
+
+@pytest.fixture
+def iac_multiple_vulns() -> str:
+    return _IAC_MULTIPLE_VULNERABILITIES
+
+
+@pytest.fixture
+def iac_no_vulns() -> str:
+    return _IAC_NO_VULNERABILITIES
```

Comment:
It looks to me that using fixtures instead of constants makes tests look more complicated (more arguments) for no benefits, am I missing something?

Reply:
This may be personal preference, but I find fixtures more elegant in tests. Should we keep the global variables, and simply remove the leading underscores from their names instead?

Reply:
Yes, I would prefer if we just removed the leading underscores. The advantage of fixtures is the ability to setup/teardown, and there is no need for this on constants.